app/controllers/offline_reservations_controller.rb

```diff
@@ -8,6 +8,7 @@ class OfflineReservationsController < ApplicationController
   before_action :init_current_facility
   before_action :load_instrument
   before_action :load_reservation, only: %i(edit update)
+  after_action :flag_ongoing_reservations_as_problem, only: %i(create)
```

Comment:
I don't love doing this in an `after_action`. I would lean towards moving it into the success block of the action or possibly an after create callback on the OfflineReservation.